tap_stripe/__init__.py

```diff
@@ -728,6 +703,54 @@ def sync_sub_stream(sub_stream_name, parent_obj, updates=False):
                 Context.new_counts[sub_stream_name] += 1
 
 
+def sync_sub_stream(child_stream, bookmark_value):
+    """
+    Get the parent records based on the bookmark and corresponding child stream records
+    """
+    # Get the parent stream of the stream
+    parent_stream = PARENT_STREAM_MAP[child_stream]
+    # Get the replication key for the stream
+    replication_key = STREAM_REPLICATION_KEY[parent_stream]
+
+    filter_key = replication_key
```

Comment:
We are not changing the value of the replication key, do we need this assignment?

Reply:
Removed the assignment